dynacred/src/Data.ts

```diff
@@ -660,17 +657,43 @@ export class Data {
         return this.parties;
     }
 
-    async updateParties(): Promise<Party[]> {
+    async fetchOrgKeys(ppi: PopPartyInstance): Promise<Point[]> {
+        const piDarc = await DarcInstance.fromByzcoin(this.bc, ppi.darcID);
+        const orgDarcs = piDarc.darc.rules.list.find((l) => l.action === "invoke:popParty.finalize").getIdentities();
+        const orgPers: Point[] = [];
+        const contacts = this.contacts.concat(this.contact);
+
+        for (const orgDarc of orgDarcs) {
+            // Remove leading "darc:" from expression
+            const orgDarcID = Buffer.from(orgDarc.substr(5), "hex");
+            const contact = contacts.find((c) => c.credentialInstance.darcID.equals(orgDarcID));
+            if (contact === undefined) {
+                return Promise.reject("didn't find organizer in contacts");
```

Comment:
Below, you're using `throw Error(...)` to mark failure. Which one between `Promise.reject` and `throw` is the one you prefer (I lean toward the second, as converting to/from `async` means less code delta)?

Reply:
I prefer good ol' `throw`, too...